[INFO] Scanning for projects...
[INFO] 
[INFO] -------------------< com.ditsolution:immobilier-api >-------------------
[INFO] Building immobilier-api 1.0.0-SNAPSHOT
[INFO]   from pom.xml
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- quarkus:3.25.4:dev (default-cli) @ immobilier-api ---
[INFO] Invoking resources:3.3.1:resources (default-resources) @ immobilier-api
[INFO] Copying 21 resources from src/main/resources to target/classes
[INFO] Invoking quarkus:3.25.4:generate-code (default) @ immobilier-api
[INFO] Invoking compiler:3.14.0:compile (default-compile) @ immobilier-api
[INFO] Recompiling the module because of added or removed source files.
[INFO] Compiling 101 source files with javac [debug parameters release 21] to target/classes
[INFO] /Users/chekna/Desktop/App Immo civ/Immo/immobilier-api/src/main/java/com/ditsolution/features/notification/service/NotificationService.java: /Users/chekna/Desktop/App Immo civ/Immo/immobilier-api/src/main/java/com/ditsolution/features/notification/service/NotificationService.java uses or overrides a deprecated API.
[INFO] /Users/chekna/Desktop/App Immo civ/Immo/immobilier-api/src/main/java/com/ditsolution/features/notification/service/NotificationService.java: Recompile with -Xlint:deprecation for details.
[INFO] /Users/chekna/Desktop/App Immo civ/Immo/immobilier-api/src/main/java/com/ditsolution/features/listing/repository/ListingRepository.java: /Users/chekna/Desktop/App Immo civ/Immo/immobilier-api/src/main/java/com/ditsolution/features/listing/repository/ListingRepository.java uses unchecked or unsafe operations.
[INFO] /Users/chekna/Desktop/App Immo civ/Immo/immobilier-api/src/main/java/com/ditsolution/features/listing/repository/ListingRepository.java: Recompile with -Xlint:unchecked for details.
[INFO] Invoking resources:3.3.1:testResources (default-testResources) @ immobilier-api
[INFO] skip non existing resourceDirectory /Users/chekna/Desktop/App Immo civ/Immo/immobilier-api/src/test/resources
[INFO] Invoking quarkus:3.25.4:generate-code-tests (default) @ immobilier-api
[INFO] Invoking compiler:3.14.0:testCompile (default-testCompile) @ immobilier-api
[INFO] No sources to compile
Listening for transport dt_socket at address: 5005
Press [e] to edit command line args (currently ''), [h] for more options>
Tests paused
Press [e] to edit command line args (currently ''), [r] to resume testing, [h] for more options>
Press [e] to edit command line args (currently ''), [r] to resume testing, [o] Toggle test output, [h] for more options>
2025-10-16 19:04:58,649 INFO  [org.tes.DockerClientFactory] (build-6) Testcontainers version: 1.21.3
2025-10-16 19:04:58,839 INFO  [org.tes.doc.DockerClientProviderStrategy] (build-6) Loaded org.testcontainers.dockerclient.UnixSocketClientProviderStrategy from ~/.testcontainers.properties, will try it first
2025-10-16 19:04:58,857 WARN  [org.tes.doc.DockerClientProviderStrategy] (build-6) DOCKER_HOST unix:///var/run/docker.sock is not listening: java.net.ConnectException: Connection refused
	at java.base/sun.nio.ch.UnixDomainSockets.connect0(Native Method)
	at java.base/sun.nio.ch.UnixDomainSockets.connect(UnixDomainSockets.java:154)
	at java.base/sun.nio.ch.UnixDomainSockets.connect(UnixDomainSockets.java:150)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:878)
	at java.base/java.nio.channels.SocketChannel.open(SocketChannel.java:285)
	at com.github.dockerjava.transport.UnixSocket.<init>(UnixSocket.java:45)
	at com.github.dockerjava.transport.UnixSocket.get(UnixSocket.java:27)
	at org.testcontainers.dockerclient.DockerClientProviderStrategy.lambda$test$2(DockerClientProviderStrategy.java:194)
	at org.testcontainers.dockerclient.DockerClientProviderStrategy.test(DockerClientProviderStrategy.java:208)
	at org.testcontainers.dockerclient.DockerClientProviderStrategy.tryOutStrategy(DockerClientProviderStrategy.java:284)
	at org.testcontainers.dockerclient.DockerClientProviderStrategy.lambda$getFirstValidStrategy$5(DockerClientProviderStrategy.java:263)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:178)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:179)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:179)
	at java.base/java.util.ArrayList$ArrayListSpliterator.tryAdvance(ArrayList.java:1685)
	at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:129)
	at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:527)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:513)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base/java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:150)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:647)
	at org.testcontainers.dockerclient.DockerClientProviderStrategy.getFirstValidStrategy(DockerClientProviderStrategy.java:264)
	at org.testcontainers.DockerClientFactory.getOrInitializeStrategy(DockerClientFactory.java:154)
	at org.testcontainers.DockerClientFactory.client(DockerClientFactory.java:196)
	at org.testcontainers.DockerClientFactory.isDockerAvailable(DockerClientFactory.java:138)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at io.quarkus.deployment.IsContainerRuntimeWorking$TestContainersStrategy.get(IsContainerRuntimeWorking.java:90)
	at io.quarkus.deployment.IsContainerRuntimeWorking$TestContainersStrategy.get(IsContainerRuntimeWorking.java:53)
	at io.quarkus.deployment.IsContainerRuntimeWorking.getAsBoolean(IsContainerRuntimeWorking.java:37)
	at io.quarkus.deployment.builditem.ContainerRuntimeStatusBuildItem.isContainerRuntimeAvailable(ContainerRuntimeStatusBuildItem.java:18)
	at io.quarkus.devservices.deployment.DevServicesProcessor.config(DevServicesProcessor.java:172)
	at java.base/java.lang.invoke.MethodHandle.invokeWithArguments(MethodHandle.java:733)
	at io.quarkus.deployment.ExtensionLoader$3.execute(ExtensionLoader.java:889)
	at io.quarkus.builder.BuildContext.run(BuildContext.java:255)
	at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2651)
	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2630)
	at org.jboss.threads.EnhancedQueueExecutor.runThreadBody(EnhancedQueueExecutor.java:1622)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1589)
	at java.base/java.lang.Thread.run(Thread.java:1583)
	at org.jboss.threads.JBossThread.run(JBossThread.java:501)

2025-10-16 19:04:58,859 WARN  [org.tes.doc.DockerClientProviderStrategy] (build-6) DOCKER_HOST unix:///Users/chekna/.docker/run/docker.sock is not listening: java.net.ConnectException: Connection refused
	at java.base/sun.nio.ch.UnixDomainSockets.connect0(Native Method)
	at java.base/sun.nio.ch.UnixDomainSockets.connect(UnixDomainSockets.java:154)
	at java.base/sun.nio.ch.UnixDomainSockets.connect(UnixDomainSockets.java:150)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:878)
	at java.base/java.nio.channels.SocketChannel.open(SocketChannel.java:285)
	at com.github.dockerjava.transport.UnixSocket.<init>(UnixSocket.java:45)
	at com.github.dockerjava.transport.UnixSocket.get(UnixSocket.java:27)
	at org.testcontainers.dockerclient.DockerClientProviderStrategy.lambda$test$2(DockerClientProviderStrategy.java:194)
	at org.testcontainers.dockerclient.DockerClientProviderStrategy.test(DockerClientProviderStrategy.java:208)
	at org.testcontainers.dockerclient.DockerClientProviderStrategy.tryOutStrategy(DockerClientProviderStrategy.java:284)
	at org.testcontainers.dockerclient.DockerClientProviderStrategy.lambda$getFirstValidStrategy$5(DockerClientProviderStrategy.java:263)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:178)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:179)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:179)
	at java.base/java.util.ArrayList$ArrayListSpliterator.tryAdvance(ArrayList.java:1685)
	at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:129)
	at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:527)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:513)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base/java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:150)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:647)
	at org.testcontainers.dockerclient.DockerClientProviderStrategy.getFirstValidStrategy(DockerClientProviderStrategy.java:264)
	at org.testcontainers.DockerClientFactory.getOrInitializeStrategy(DockerClientFactory.java:154)
	at org.testcontainers.DockerClientFactory.client(DockerClientFactory.java:196)
	at org.testcontainers.DockerClientFactory.isDockerAvailable(DockerClientFactory.java:138)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at io.quarkus.deployment.IsContainerRuntimeWorking$TestContainersStrategy.get(IsContainerRuntimeWorking.java:90)
	at io.quarkus.deployment.IsContainerRuntimeWorking$TestContainersStrategy.get(IsContainerRuntimeWorking.java:53)
	at io.quarkus.deployment.IsContainerRuntimeWorking.getAsBoolean(IsContainerRuntimeWorking.java:37)
	at io.quarkus.deployment.builditem.ContainerRuntimeStatusBuildItem.isContainerRuntimeAvailable(ContainerRuntimeStatusBuildItem.java:18)
	at io.quarkus.devservices.deployment.DevServicesProcessor.config(DevServicesProcessor.java:172)
	at java.base/java.lang.invoke.MethodHandle.invokeWithArguments(MethodHandle.java:733)
	at io.quarkus.deployment.ExtensionLoader$3.execute(ExtensionLoader.java:889)
	at io.quarkus.builder.BuildContext.run(BuildContext.java:255)
	at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2651)
	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2630)
	at org.jboss.threads.EnhancedQueueExecutor.runThreadBody(EnhancedQueueExecutor.java:1622)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1589)
	at java.base/java.lang.Thread.run(Thread.java:1583)
	at org.jboss.threads.JBossThread.run(JBossThread.java:501)

2025-10-16 19:04:58,861 INFO  [org.tes.doc.DockerMachineClientProviderStrategy] (build-6) docker-machine executable was not found on PATH ([/Users/chekna/.local/bin, /Users/chekna/.nvm/versions/node/v22.12.0/bin, /opt/homebrew/bin, /opt/homebrew/sbin, /usr/local/bin, /System/Cryptexes/App/usr/bin, /usr/bin, /bin, /usr/sbin, /sbin, /var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin, /var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin, /var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin, /Library/Apple/usr/bin, /Users/chekna/.local/bin, /Users/chekna/.nvm/versions/node/v22.12.0/bin])
2025-10-16 19:04:58,862 ERROR [org.tes.doc.DockerClientProviderStrategy] (build-6) Could not find a valid Docker environment. Please check configuration. Attempted configurations were:
As no valid configuration was found, execution cannot continue.
See https://java.testcontainers.org/on_failure.html for more details.
2025-10-16 19:04:59,642 WARN  [io.qua.dep.uti.ContainerRuntimeUtil] (build-6) Command "docker" failed. Rootless container runtime detection might not be reliable or the container service is not running at all.
2025-10-16 19:05:00,370 INFO  [io.und.websockets] (Quarkus Main Thread) UT026003: Adding annotated server endpoint class com.ditsolution.features.messaging.websocket.MessageWebSocket for path /ws/messages/{token}
2025-10-16 19:05:00,488 WARN  [io.qua.config] (Quarkus Main Thread) Unrecognized configuration key "quarkus.test.containers.enabled" was provided; it will be ignored; verify that the dependency extension for this configuration is set or that you did not make a typo
2025-10-16 19:05:00,488 WARN  [io.qua.config] (Quarkus Main Thread) Unrecognized configuration key "quarkus.jackson.date-format" was provided; it will be ignored; verify that the dependency extension for this configuration is set or that you did not make a typo
2025-10-16 19:05:00,494 WARN  [io.qua.config] (Quarkus Main Thread) The "quarkus.hibernate-orm.database.generation" config property is deprecated and should not be used anymore.
__  ____  __  _____   ___  __ ____  ______ 
 --/ __ \/ / / / _ | / _ \/ //_/ / / / __/ 
 -/ /_/ / /_/ / __ |/ , _/ ,< / /_/ /\ \   
--\___\_\____/_/ |_/_/|_/_/|_|\____/___/   
2025-10-16 19:05:00,562 WARN  [io.sma.config] (Quarkus Main Thread) SRCFG01008: The value OPTIONAL has been converted by a Boolean Converter to "false"
2025-10-16 19:05:00,688 WARN  [io.agr.pool] (agroal-11) Datasource '<default>': FATAL: password authentication failed for user "immobilier_user"
2025-10-16 19:05:00,695 ERROR [io.qua.run.Application] (Quarkus Main Thread) Failed to start application: java.lang.RuntimeException: Failed to start quarkus
	at io.quarkus.runner.ApplicationImpl.doStart(Unknown Source)
	at io.quarkus.runtime.Application.start(Application.java:101)
	at io.quarkus.runtime.ApplicationLifecycleManager.run(ApplicationLifecycleManager.java:119)
	at io.quarkus.runtime.Quarkus.run(Quarkus.java:80)
	at io.quarkus.runtime.Quarkus.run(Quarkus.java:51)
	at io.quarkus.runtime.Quarkus.run(Quarkus.java:144)
	at io.quarkus.runner.GeneratedMain.main(Unknown Source)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at io.quarkus.runner.bootstrap.StartupActionImpl$1.run(StartupActionImpl.java:135)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: org.flywaydb.core.internal.exception.sqlExceptions.FlywaySqlUnableToConnectToDbException: Unable to obtain connection from database: FATAL: password authentication failed for user "immobilier_user"
-----------------------------------------------------------------------------------------------------------
SQL State  : 28P01
Error Code : 0
Message    : FATAL: password authentication failed for user "immobilier_user"

	at org.flywaydb.core.internal.jdbc.JdbcUtils.openConnection(JdbcUtils.java:70)
	at org.flywaydb.core.internal.jdbc.JdbcConnectionFactory.<init>(JdbcConnectionFactory.java:76)
	at org.flywaydb.core.FlywayExecutor.execute(FlywayExecutor.java:136)
	at org.flywaydb.core.Flyway.migrate(Flyway.java:188)
	at io.quarkus.flyway.runtime.FlywayRecorder.doStartActions(FlywayRecorder.java:150)
	at io.quarkus.runner.recorded.FlywayProcessor$startActions1575138505.deploy_0(Unknown Source)
	at io.quarkus.runner.recorded.FlywayProcessor$startActions1575138505.deploy(Unknown Source)
	... 11 more
Caused by: org.postgresql.util.PSQLException: FATAL: password authentication failed for user "immobilier_user"
	at org.postgresql.core.v3.ConnectionFactoryImpl.doAuthentication(ConnectionFactoryImpl.java:777)
	at org.postgresql.core.v3.ConnectionFactoryImpl.tryConnect(ConnectionFactoryImpl.java:234)
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:289)
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:57)
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:277)
	at org.postgresql.Driver.makeConnection(Driver.java:448)
	at org.postgresql.Driver.connect(Driver.java:298)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:225)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:585)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:566)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	... 1 more

Press [space] to restart, [e] to edit command line args (currently ''), [r] to resume testing, [o] Toggle test output, [h] for more options>
2025-10-16 19:05:00,723 WARN  [io.qua.config] (main) Unrecognized configuration key "quarkus.test.containers.enabled" was provided; it will be ignored; verify that the dependency extension for this configuration is set or that you did not make a typo
2025-10-16 19:05:00,723 WARN  [io.qua.config] (main) Unrecognized configuration key "quarkus.test.containers.enabled" was provided; it will be ignored; verify that the dependency extension for this configuration is set or that you did not make a typo
2025-10-16 19:05:00,724 WARN  [io.qua.config] (main) Unrecognized configuration key "quarkus.jackson.date-format" was provided; it will be ignored; verify that the dependency extension for this configuration is set or that you did not make a typo
2025-10-16 19:05:00,724 WARN  [io.qua.config] (main) Unrecognized configuration key "quarkus.jackson.date-format" was provided; it will be ignored; verify that the dependency extension for this configuration is set or that you did not make a typo
2025-10-16 19:05:00,724 INFO  [io.qua.dep.dev.IsolatedDevModeMain] (main) Attempting to start live reload endpoint to recover from previous Quarkus startup failure
2025-10-16 19:05:00,724 INFO  [io.qua.dep.dev.IsolatedDevModeMain] (main) Attempting to start live reload endpoint to recover from previous Quarkus startup failure
Press [e] to edit command line args (currently ''), [r] to resume testing, [o] Toggle test output, [h] for more options>
2025-10-16 19:05:50,973 INFO  [org.tes.DockerClientFactory] (build-37) Testcontainers version: 1.21.3
2025-10-16 19:05:51,002 INFO  [org.tes.doc.DockerClientProviderStrategy] (build-37) Loaded org.testcontainers.dockerclient.UnixSocketClientProviderStrategy from ~/.testcontainers.properties, will try it first
2025-10-16 19:05:51,003 WARN  [org.tes.doc.DockerClientProviderStrategy] (build-37) DOCKER_HOST unix:///var/run/docker.sock is not listening: java.net.ConnectException: Connection refused
	at java.base/sun.nio.ch.UnixDomainSockets.connect0(Native Method)
	at java.base/sun.nio.ch.UnixDomainSockets.connect(UnixDomainSockets.java:154)
	at java.base/sun.nio.ch.UnixDomainSockets.connect(UnixDomainSockets.java:150)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:878)
	at java.base/java.nio.channels.SocketChannel.open(SocketChannel.java:285)
	at com.github.dockerjava.transport.UnixSocket.<init>(UnixSocket.java:45)
	at com.github.dockerjava.transport.UnixSocket.get(UnixSocket.java:27)
	at org.testcontainers.dockerclient.DockerClientProviderStrategy.lambda$test$2(DockerClientProviderStrategy.java:194)
	at org.testcontainers.dockerclient.DockerClientProviderStrategy.test(DockerClientProviderStrategy.java:208)
	at org.testcontainers.dockerclient.DockerClientProviderStrategy.tryOutStrategy(DockerClientProviderStrategy.java:284)
	at org.testcontainers.dockerclient.DockerClientProviderStrategy.lambda$getFirstValidStrategy$5(DockerClientProviderStrategy.java:263)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:178)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:179)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:179)
	at java.base/java.util.ArrayList$ArrayListSpliterator.tryAdvance(ArrayList.java:1685)
	at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:129)
	at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:527)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:513)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base/java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:150)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:647)
	at org.testcontainers.dockerclient.DockerClientProviderStrategy.getFirstValidStrategy(DockerClientProviderStrategy.java:264)
	at org.testcontainers.DockerClientFactory.getOrInitializeStrategy(DockerClientFactory.java:154)
	at org.testcontainers.DockerClientFactory.client(DockerClientFactory.java:196)
	at org.testcontainers.DockerClientFactory.isDockerAvailable(DockerClientFactory.java:138)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at io.quarkus.deployment.IsContainerRuntimeWorking$TestContainersStrategy.get(IsContainerRuntimeWorking.java:90)
	at io.quarkus.deployment.IsContainerRuntimeWorking$TestContainersStrategy.get(IsContainerRuntimeWorking.java:53)
	at io.quarkus.deployment.IsContainerRuntimeWorking.getAsBoolean(IsContainerRuntimeWorking.java:37)
	at io.quarkus.deployment.builditem.ContainerRuntimeStatusBuildItem.isContainerRuntimeAvailable(ContainerRuntimeStatusBuildItem.java:18)
	at io.quarkus.devservices.deployment.DevServicesProcessor.config(DevServicesProcessor.java:172)
	at java.base/java.lang.invoke.MethodHandle.invokeWithArguments(MethodHandle.java:733)
	at io.quarkus.deployment.ExtensionLoader$3.execute(ExtensionLoader.java:889)
	at io.quarkus.builder.BuildContext.run(BuildContext.java:255)
	at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2651)
	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2630)
	at org.jboss.threads.EnhancedQueueExecutor.runThreadBody(EnhancedQueueExecutor.java:1622)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1589)
	at java.base/java.lang.Thread.run(Thread.java:1583)
	at org.jboss.threads.JBossThread.run(JBossThread.java:501)

2025-10-16 19:05:51,003 WARN  [org.tes.doc.DockerClientProviderStrategy] (build-37) DOCKER_HOST unix:///Users/chekna/.docker/run/docker.sock is not listening: java.net.ConnectException: Connection refused
	at java.base/sun.nio.ch.UnixDomainSockets.connect0(Native Method)
	at java.base/sun.nio.ch.UnixDomainSockets.connect(UnixDomainSockets.java:154)
	at java.base/sun.nio.ch.UnixDomainSockets.connect(UnixDomainSockets.java:150)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:878)
	at java.base/java.nio.channels.SocketChannel.open(SocketChannel.java:285)
	at com.github.dockerjava.transport.UnixSocket.<init>(UnixSocket.java:45)
	at com.github.dockerjava.transport.UnixSocket.get(UnixSocket.java:27)
	at org.testcontainers.dockerclient.DockerClientProviderStrategy.lambda$test$2(DockerClientProviderStrategy.java:194)
	at org.testcontainers.dockerclient.DockerClientProviderStrategy.test(DockerClientProviderStrategy.java:208)
	at org.testcontainers.dockerclient.DockerClientProviderStrategy.tryOutStrategy(DockerClientProviderStrategy.java:284)
	at org.testcontainers.dockerclient.DockerClientProviderStrategy.lambda$getFirstValidStrategy$5(DockerClientProviderStrategy.java:263)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:178)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:179)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:179)
	at java.base/java.util.ArrayList$ArrayListSpliterator.tryAdvance(ArrayList.java:1685)
	at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:129)
	at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:527)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:513)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base/java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:150)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:647)
	at org.testcontainers.dockerclient.DockerClientProviderStrategy.getFirstValidStrategy(DockerClientProviderStrategy.java:264)
	at org.testcontainers.DockerClientFactory.getOrInitializeStrategy(DockerClientFactory.java:154)
	at org.testcontainers.DockerClientFactory.client(DockerClientFactory.java:196)
	at org.testcontainers.DockerClientFactory.isDockerAvailable(DockerClientFactory.java:138)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at io.quarkus.deployment.IsContainerRuntimeWorking$TestContainersStrategy.get(IsContainerRuntimeWorking.java:90)
	at io.quarkus.deployment.IsContainerRuntimeWorking$TestContainersStrategy.get(IsContainerRuntimeWorking.java:53)
	at io.quarkus.deployment.IsContainerRuntimeWorking.getAsBoolean(IsContainerRuntimeWorking.java:37)
	at io.quarkus.deployment.builditem.ContainerRuntimeStatusBuildItem.isContainerRuntimeAvailable(ContainerRuntimeStatusBuildItem.java:18)
	at io.quarkus.devservices.deployment.DevServicesProcessor.config(DevServicesProcessor.java:172)
	at java.base/java.lang.invoke.MethodHandle.invokeWithArguments(MethodHandle.java:733)
	at io.quarkus.deployment.ExtensionLoader$3.execute(ExtensionLoader.java:889)
	at io.quarkus.builder.BuildContext.run(BuildContext.java:255)
	at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2651)
	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2630)
	at org.jboss.threads.EnhancedQueueExecutor.runThreadBody(EnhancedQueueExecutor.java:1622)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1589)
	at java.base/java.lang.Thread.run(Thread.java:1583)
	at org.jboss.threads.JBossThread.run(JBossThread.java:501)

2025-10-16 19:05:51,004 INFO  [org.tes.doc.DockerMachineClientProviderStrategy] (build-37) docker-machine executable was not found on PATH ([/Users/chekna/.local/bin, /Users/chekna/.nvm/versions/node/v22.12.0/bin, /opt/homebrew/bin, /opt/homebrew/sbin, /usr/local/bin, /System/Cryptexes/App/usr/bin, /usr/bin, /bin, /usr/sbin, /sbin, /var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin, /var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin, /var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin, /Library/Apple/usr/bin, /Users/chekna/.local/bin, /Users/chekna/.nvm/versions/node/v22.12.0/bin])
2025-10-16 19:05:51,004 ERROR [org.tes.doc.DockerClientProviderStrategy] (build-37) Could not find a valid Docker environment. Please check configuration. Attempted configurations were:
As no valid configuration was found, execution cannot continue.
See https://java.testcontainers.org/on_failure.html for more details.
2025-10-16 19:05:51,777 WARN  [io.qua.dep.uti.ContainerRuntimeUtil] (build-37) Command "docker" failed. Rootless container runtime detection might not be reliable or the container service is not running at all.
2025-10-16 19:05:51,993 INFO  [io.und.websockets] (Quarkus Main Thread) UT026003: Adding annotated server endpoint class com.ditsolution.features.messaging.websocket.MessageWebSocket for path /ws/messages/{token}
2025-10-16 19:05:52,035 WARN  [io.qua.config] (Quarkus Main Thread) Unrecognized configuration key "quarkus.test.containers.enabled" was provided; it will be ignored; verify that the dependency extension for this configuration is set or that you did not make a typo
2025-10-16 19:05:52,035 WARN  [io.qua.config] (Quarkus Main Thread) Unrecognized configuration key "quarkus.jackson.date-format" was provided; it will be ignored; verify that the dependency extension for this configuration is set or that you did not make a typo
2025-10-16 19:05:52,038 WARN  [io.qua.config] (Quarkus Main Thread) The "quarkus.hibernate-orm.database.generation" config property is deprecated and should not be used anymore.
__  ____  __  _____   ___  __ ____  ______ 
 --/ __ \/ / / / _ | / _ \/ //_/ / / / __/ 
 -/ /_/ / /_/ / __ |/ , _/ ,< / /_/ /\ \   
--\___\_\____/_/ |_/_/|_/_/|_|\____/___/   
2025-10-16 19:05:52,040 WARN  [io.sma.config] (Quarkus Main Thread) SRCFG01008: The value OPTIONAL has been converted by a Boolean Converter to "false"
2025-10-16 19:05:52,063 WARN  [io.agr.pool] (agroal-21) Datasource '<default>': FATAL: password authentication failed for user "immobilier_user"
2025-10-16 19:05:52,064 ERROR [io.qua.run.Application] (Quarkus Main Thread) Failed to start application: java.lang.RuntimeException: Failed to start quarkus
	at io.quarkus.runner.ApplicationImpl.doStart(Unknown Source)
	at io.quarkus.runtime.Application.start(Application.java:101)
	at io.quarkus.runtime.ApplicationLifecycleManager.run(ApplicationLifecycleManager.java:119)
	at io.quarkus.runtime.Quarkus.run(Quarkus.java:80)
	at io.quarkus.runtime.Quarkus.run(Quarkus.java:51)
	at io.quarkus.runtime.Quarkus.run(Quarkus.java:144)
	at io.quarkus.runner.GeneratedMain.main(Unknown Source)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at io.quarkus.runner.bootstrap.StartupActionImpl$1.run(StartupActionImpl.java:135)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: org.flywaydb.core.internal.exception.sqlExceptions.FlywaySqlUnableToConnectToDbException: Unable to obtain connection from database: FATAL: password authentication failed for user "immobilier_user"
-----------------------------------------------------------------------------------------------------------
SQL State  : 28P01
Error Code : 0
Message    : FATAL: password authentication failed for user "immobilier_user"

	at org.flywaydb.core.internal.jdbc.JdbcUtils.openConnection(JdbcUtils.java:70)
	at org.flywaydb.core.internal.jdbc.JdbcConnectionFactory.<init>(JdbcConnectionFactory.java:76)
	at org.flywaydb.core.FlywayExecutor.execute(FlywayExecutor.java:136)
	at org.flywaydb.core.Flyway.migrate(Flyway.java:188)
	at io.quarkus.flyway.runtime.FlywayRecorder.doStartActions(FlywayRecorder.java:150)
	at io.quarkus.runner.recorded.FlywayProcessor$startActions1575138505.deploy_0(Unknown Source)
	at io.quarkus.runner.recorded.FlywayProcessor$startActions1575138505.deploy(Unknown Source)
	... 11 more
Caused by: org.postgresql.util.PSQLException: FATAL: password authentication failed for user "immobilier_user"
	at org.postgresql.core.v3.ConnectionFactoryImpl.doAuthentication(ConnectionFactoryImpl.java:777)
	at org.postgresql.core.v3.ConnectionFactoryImpl.tryConnect(ConnectionFactoryImpl.java:234)
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:289)
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:57)
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:277)
	at org.postgresql.Driver.makeConnection(Driver.java:448)
	at org.postgresql.Driver.connect(Driver.java:298)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:225)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:585)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:566)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	... 1 more

2025-10-16 19:05:52,082 WARN  [io.qua.config] (vert.x-worker-thread-2) Unrecognized configuration key "quarkus.test.containers.enabled" was provided; it will be ignored; verify that the dependency extension for this configuration is set or that you did not make a typo
2025-10-16 19:05:52,082 WARN  [io.qua.config] (vert.x-worker-thread-2) Unrecognized configuration key "quarkus.jackson.date-format" was provided; it will be ignored; verify that the dependency extension for this configuration is set or that you did not make a typo
2025-10-16 19:05:52,082 ERROR [io.qua.dep.dev.IsolatedDevModeMain] (vert.x-worker-thread-2) Failed to start quarkus: io.quarkus.dev.appstate.ApplicationStartException: java.lang.RuntimeException: Failed to start quarkus
	at io.quarkus.dev.appstate.ApplicationStateNotification.waitForApplicationStart(ApplicationStateNotification.java:63)
	at io.quarkus.runner.bootstrap.StartupActionImpl.runMainClass(StartupActionImpl.java:161)
	at io.quarkus.deployment.dev.IsolatedDevModeMain.restartApp(IsolatedDevModeMain.java:206)
	at io.quarkus.deployment.dev.IsolatedDevModeMain.restartCallback(IsolatedDevModeMain.java:187)
	at io.quarkus.deployment.dev.RuntimeUpdatesProcessor.doScan(RuntimeUpdatesProcessor.java:555)
	at io.quarkus.deployment.dev.RuntimeUpdatesProcessor.doScan(RuntimeUpdatesProcessor.java:455)
	at io.quarkus.vertx.http.runtime.devmode.VertxHttpHotReplacementSetup$6.call(VertxHttpHotReplacementSetup.java:163)
	at io.quarkus.vertx.http.runtime.devmode.VertxHttpHotReplacementSetup$6.call(VertxHttpHotReplacementSetup.java:150)
	at io.vertx.core.impl.ContextImpl.lambda$executeBlocking$4(ContextImpl.java:192)
	at io.vertx.core.impl.ContextInternal.dispatch(ContextInternal.java:270)
	at io.vertx.core.impl.ContextImpl$1.execute(ContextImpl.java:221)
	at io.vertx.core.impl.WorkerTask.run(WorkerTask.java:56)
	at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2651)
	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2630)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1586)
	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: java.lang.RuntimeException: Failed to start quarkus
	at io.quarkus.runner.ApplicationImpl.doStart(Unknown Source)
	at io.quarkus.runtime.Application.start(Application.java:101)
	at io.quarkus.runtime.ApplicationLifecycleManager.run(ApplicationLifecycleManager.java:119)
	at io.quarkus.runtime.Quarkus.run(Quarkus.java:80)
	at io.quarkus.runtime.Quarkus.run(Quarkus.java:51)
	at io.quarkus.runtime.Quarkus.run(Quarkus.java:144)
	at io.quarkus.runner.GeneratedMain.main(Unknown Source)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at io.quarkus.runner.bootstrap.StartupActionImpl$1.run(StartupActionImpl.java:135)
	... 1 more
Caused by: org.flywaydb.core.internal.exception.sqlExceptions.FlywaySqlUnableToConnectToDbException: Unable to obtain connection from database: FATAL: password authentication failed for user "immobilier_user"
-----------------------------------------------------------------------------------------------------------
SQL State  : 28P01
Error Code : 0
Message    : FATAL: password authentication failed for user "immobilier_user"

	at org.flywaydb.core.internal.jdbc.JdbcUtils.openConnection(JdbcUtils.java:70)
	at org.flywaydb.core.internal.jdbc.JdbcConnectionFactory.<init>(JdbcConnectionFactory.java:76)
	at org.flywaydb.core.FlywayExecutor.execute(FlywayExecutor.java:136)
	at org.flywaydb.core.Flyway.migrate(Flyway.java:188)
	at io.quarkus.flyway.runtime.FlywayRecorder.doStartActions(FlywayRecorder.java:150)
	at io.quarkus.runner.recorded.FlywayProcessor$startActions1575138505.deploy_0(Unknown Source)
	at io.quarkus.runner.recorded.FlywayProcessor$startActions1575138505.deploy(Unknown Source)
	... 11 more
Caused by: org.postgresql.util.PSQLException: FATAL: password authentication failed for user "immobilier_user"
	at org.postgresql.core.v3.ConnectionFactoryImpl.doAuthentication(ConnectionFactoryImpl.java:777)
	at org.postgresql.core.v3.ConnectionFactoryImpl.tryConnect(ConnectionFactoryImpl.java:234)
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:289)
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:57)
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:277)
	at org.postgresql.Driver.makeConnection(Driver.java:448)
	at org.postgresql.Driver.connect(Driver.java:298)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:225)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:585)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:566)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	... 1 more

2025-10-16 19:05:52,085 INFO  [io.qua.dep.dev.RuntimeUpdatesProcessor] (vert.x-worker-thread-2) Live reload total time: 1.453s 
Debugger failed to attach: handshake failed - connection prematurally closed
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  22:38 min
[INFO] Finished at: 2025-10-16T19:27:29+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal io.quarkus.platform:quarkus-maven-plugin:3.25.4:dev (default-cli) on project immobilier-api: Failed to run: Dev mode process did not complete successfully -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
